lib/src/working_copy.rs

```diff
@@ -258,6 +258,8 @@ pub enum UntrackedReason {
         /// Maximum allowed size.
         max_size: u64,
     },
+    /// Setting snapshot.auto-track did not match this file.
```

Comment:
nit: I initially parsed "Setting" as a verb. Maybe "Setting" -> "Config"?

Reply:
How about: "File does not match the fileset specified in snapshot.auto-track."?